include/z64actor.h

```diff
@@ -171,12 +171,12 @@ typedef struct Actor {
     /* 0x085 */ u8 floorBgId; // Bg ID of the floor polygon directly below the actor
     /* 0x086 */ s16 wallYaw; // Y rotation of the wall polygon the actor is touching
     /* 0x088 */ f32 floorHeight; // Y position of the floor polygon directly below the actor
-    /* 0x08C */ f32 yDistToWater; // Distance to the surface of active waterbox. Negative value means above water
+    /* 0x08C */ f32 yDirDistToWater; // Distance to the surface of active waterbox. Negative value means above water
```

Comment:
```suggestion
    /* 0x08C */ f32 yDirDistToWater; // Directional Distance to the surface of active waterbox. Negative value means above water
```
I would add the word "Directional" Since I could think of a few abbreviations `Dir` could mean, even if it's obvious looking into context

Reply:
I still think `depthIntoWater` or similar is a better name for this: it avoids having to look at the comment altogether

Reply:
```suggestion
    /* 0x08C */ f32 yDirDistToWater; // Directed Distance to the surface of active waterbox. Negative value means water is below.
```